src/Tribe/Views/V2/View.php

```diff
@@ -1765,28 +1765,23 @@ protected function on_page_reset() {
 	protected function get_breadcrumbs() {
 		$context     = $this->context;
 		$breadcrumbs = [];
-		$context_tax = $context->get( TEC::TAXONOMY, false );
+		$taxonomy    = TEC::TAXONOMY;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f91eb646d66021433c8be82dc7d8ad3a</sub>